spec/views/posts/show.html.erb_spec.rb

```diff
@@ -1,5 +1,49 @@
 require 'rails_helper'
+RSpec.describe 'Renders the post show page', type: :feature do
+  before :each do
+    @user = User.create(name: 'Marc', photo: 'https://media.gettyimages.com/id/1278139568/nl/foto/studioportret-van-20-jaar-oude-vrouw.jpg?s=612x612&w=0&k=20&c=3Bd4Ot79Z1ZKoCwAl0qFQ9hoBrQTar4SqtPefHOBEkg=',
+                        bio: 'Teacher from Brazil.', posts_counter: 5)
 
-RSpec.describe 'posts/show.html.erb', type: :view do
-  pending "add some examples to (or delete) #{__FILE__}"
+    @first_post = Post.create(title: 'Hello there', text: 'This is my first post',
+                              comments_counter: 0, likes_counter: 0, author_id: @user.id)
+    Comment.create(text: 'Hi Marc!', author_id: @user.id, post_id: @first_post.id)
+    Comment.create(text: 'How are you?', author_id: @user.id, post_id: @first_post.id)
+    Comment.create(text: 'Awesome Bro', author_id: @user.id, post_id: @first_post.id)
+
+    @first_post.update(comments_counter: @first_post.comments.count)
+    visit user_posts_path(@first_post.author, @first_post)
+  end
+
+  scenario 'displays the post title' do
+    visit user_posts_path(@first_post.author, @first_post)
+    expect(page).to have_content(@first_post.title)
+  end
+
```

Comment:
- [ ] Kindly add test that `Tests that asserts you can see who wrote the post.`,  This was requested by the previous reviewer, please add it  👍🏿

Reply:
the reviewer means something like:
`expect(page).to have_content(@first_post.author.name)`